thirdparty/README.md

```diff
@@ -181,8 +181,7 @@ Files extracted from upstream source:
 
 Patches:
 
-- `0001-mingw-pragma.patch` ([GH-83452](https://github.com/godotengine/godot/pull/83452))
-- `0002-win7-8-dynamic-load.patch` ([GH-88496](https://github.com/godotengine/godot/pull/88496))
+- `0001-win7-8-dynamic-load.patch` ([GH-88496](https://github.com/godotengine/godot/pull/88496))
```

Comment:
This could likely be dropped now that we target Windows 10 (`0x0a00`) as `WINVER`.

On the other hand it's not a big change and we might want to keep it in case some users still have custom forks of Godot restoring Win7/8 support, which us dropping in 4.5 was met with mixed reception. Or if it's actually needed for tools like https://github.com/YuZhouRen86/VxKex-NEXT to properly emulate our Win10 binaries on Win7.

CC @bruvzg